### PR TITLE
Add dem0n.is-a.dev

### PR DESCRIPTION
**Pull request by probablydem0n**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [ ] You have completed your website. 
- [x] The website is reachable.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.  

## Website Link/Preview

brave_lcAQfTnikb

### CODE REVIEW

**Comment by probablydem0n** [comment]
ill complete it soon

**Comment by cupglassDEV** [comment]
This aint enough, maybe you can tell something about yourself

File: domains/dem0n.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "probablydem0n",
+        "discord": 935272776586129419
```
Comment (cupglassDEV) [inline domains/dem0n.json:4]:
```suggestion
        "discord": "935272776586129419"
```

Comment (probablydem0n) [reply domains/dem0n.json:4]: sorry for the confusion, the numbers are my discord id incase I change my username (have done many times before) the discord id will not change and will always link to my latest username, so you can contact me as for telling about myself, do you want me to add that as a string in the json file or on the website instead? I was planning on doing that on the website.

Comment (cupglassDEV) [reply domains/dem0n.json:4]: No, what i mant for yourself is, the website.

Comment (probablydem0n) [reply domains/dem0n.json:4]: Ah ok

**Pull request merged**